metadata-ingestion/src/datahub/sql_parsing/sql_parsing_aggregator.py

```diff
@@ -222,6 +224,9 @@ class SqlAggregatorReport(Report):
     sql_parsing_timer: PerfTimer = dataclasses.field(default_factory=PerfTimer)
     sql_fingerprinting_timer: PerfTimer = dataclasses.field(default_factory=PerfTimer)
     sql_formatting_timer: PerfTimer = dataclasses.field(default_factory=PerfTimer)
+    sql_parsing_cache_stats: dict = dataclasses.field(default_factory=dict)
+    parse_statement_cache_stats: dict = dataclasses.field(default_factory=dict)
+    format_query_cache_stats: dict = dataclasses.field(default_factory=dict)
```

Comment:
in the future - we probably want this instead
```suggestion
    format_query_cache_stats: Optional[dict] = dataclasses.field(default=None)
```